style.css

```diff
@@ -6,3 +6,150 @@
   color: #2c3e50;
   margin-top: 60px;
 }
+.container {
+  max-width: 1330px;
+  padding: 0 25px;
+  width: 100%;
+  margin: 0 auto;
+}
+h1, h2, h3, h4, h5 {
+  font-family: 'Roboto Mono', monospace;
+}
+.main-wrapper h1 {
+  margin: 0 0 30px;
+  font-size: 40px;
+}
+.main-wrapper h1, .main-wrapper .search-bar-wrapper {
+  text-align: center;
+}
+.main-wrapper .search-bar-wrapper {
+  margin: 0 auto 30px;
+  max-width: 300px;
+  width: 100%;
+  position: relative;
+}
+.main-wrapper .search-bar-wrapper input {
+  padding: 10px;
+  font-size: 16px;
+  border: 1px solid #000;
+  border-radius: 0;
+  width: 100%;
+  height: 100%;
+  text-overflow: ellipsis;
+  overflow: hidden;
+  background-image: url('images/search-icon.svg');
+  background-size: 20px;
+  background-position: calc(100% - 10px) center;
+  background-repeat: no-repeat;
+}
+.main-wrapper .search-bar-wrapper input:focus::placeholder {
+  color: transparent;
+}
+.main-wrapper .search-bar-wrapper input:focus {
+  border: 1px solid #000;
+  outline: none;
+}
+.main-wrapper .movie-grid {
+  display: flex;
+  flex-wrap: wrap;
+  margin: 0 -10px -20px;
+}
+.main-wrapper .movie-grid .movie-grid-item {
+  flex: 0 0 33.33%;
+  max-width: 33.33%;
+  padding: 0 10px 20px;
+}
+.main-wrapper .movie-grid .movie-grid-item .movie-grid-inner {
+  position: relative;
+  cursor: pointer;
+}
+
+.main-wrapper .movie-grid .movie-grid-item img {
+  width: 100%;
+  max-width: 100%;
+  height: auto;
+  border-radius: 20px;
+}
+.main-wrapper .movie-grid .movie-grid-item .movie-grid-inner:hover .movie-content{
```

Comment:
The class names are getting way too specific. Please try to keep the specificity under 0,3,0 - to check the specificity of the class hover over the selector to see it in the vs code. The specificity of the current class is 0,6,0 

![Screenshot (113)](https://user-images.githubusercontent.com/85559412/183240478-447710b2-fd3b-4f34-ad90-03ce7c1b8dba.png)